evalml/objectives/standard_metrics.py

```diff
@@ -838,9 +838,9 @@ def objective_function(self, y_true, y_predicted, X=None, sample_weight=None):
                 "targets contain the value 0.",
             )
         if isinstance(y_true, pd.Series):
-            y_true = y_true.values
```

Comment:
this change was needed because `.values` doesn't produce a numpy array when used with nullable. `Int64` would produce an `<IntegerArray>`, which then will be problematic for the `.mean` call below. Changing to `to_numpy` fixed this, so I'm not calling it an integer nullable incompatibility since the workaround doesnt need to worry about the types.